e2e/Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN mkdir submodules && \
   mv poa-bridge-contracts submodules && \
   cd submodules/poa-bridge-contracts && \
   git fetch && \
-  git checkout 2.2.0
+  git checkout 2.3.1
```

Comment:
Do you need update the env files to deploy contracts since `2.2.0` and `2.3.1` have different set of fields?